tests/integration/vxlan_test.py

```diff
@@ -22,16 +22,19 @@
 import pytest
 
 import libnmstate
-
 from libnmstate.error import NmstateVerificationError
 from libnmstate.schema import Interface
 
 from .testlib import assertlib
 from .testlib.bondlib import bond_interface
+from .testlib.cmd import exec_cmd
+from .testlib.cmd import RC_SUCCESS
+from .testlib.cmd import format_exec_cmd_result
 from .testlib.vxlan import VxlanState
 from .testlib.vxlan import vxlan_interfaces
 from .testlib.vxlan import vxlans_up
 from .testlib.vxlan import vxlans_down
+from .testlib.vxlan import vxlans_absent
```

Comment:
We want to keep it sorted. It is already unsorted :( , but we can place it in a better place above already.